src/fidesops/service/connectors/query_config.py

```diff
@@ -666,15 +664,47 @@ def generate_query(
         for string_path, reference_values in filtered_data.items():
             for value in reference_values:
                 request_params.append(
-                    self.prepare_params(current_request, {string_path: value})
+                    self.prepare_query_params(current_request, {string_path: value})
                 )
         logger.info(f"Populated request params for {current_request.path}")
         return request_params
 
+    @staticmethod
+    def prepare_update_params(
+        request: SaaSRequest, param_values: Dict[str, Any], body: Dict[str, Any]
+    ) -> SaaSRequestParams:
+        """
+        Populates the placeholders in the request with the given param values
+        """
+        path: str = request.path
+        params: Dict[str, Any] = {}
+
+        for param in request.request_params:
+            if param.type == "query":
+                if param.default_value:
+                    params[param.name] = param.default_value
+                elif param.references:
+                    params[param.name] = pydash.get(
+                        param_values, param.references[0].field
+                    )
+                elif param.identity:
+                    params[param.name] = pydash.get(param_values, param.identity)
+            elif param.type == "path":
+                path = path.replace(
+                    f"<{param.name}>",
+                    pydash.get(param_values, param.references[0].field),
+                )
+
+        return ("PUT", path, params, json.dumps(body))
+
     def generate_update_stmt(
         self, row: Row, policy: Policy, request: PrivacyRequest
-    ) -> Optional[List[SaaSRequestParams]]:
-        return None
+    ) -> SaaSRequestParams:
+        update_value_map: Dict[str, Any] = self.update_value_map(row, policy, request)
+        body = paths_to_dict(update_value_map)
+        current_request = self.get_request_by_action("update")
+        collection_name = self.node.address.collection
```

Comment:
Can you add some missing type hints for clarity?

Reply:
Added!